hash/98603.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0"?>
+<!DOCTYPE softwarelist SYSTEM "softwarelist.dtd">
+
+<softwarelist name="98603" description="ROM images for HP9000/300 computer">
+
+	<software name="basic51">
+		<description>HP BASIC 5.1</description>
+		<year>1988</year>
+		<publisher>HP</publisher>
+		<part name="basic51" interface="98603_cart">
+			<feature name="address" value="0x100000"/>
+			<dataarea name="rom" width="16" endianness="big" size="1048576">
+				<rom name="U1.BIN" size="65536" crc="ba50d9e0" sha1="064e3cbbc7ee01f279a289684616ae3b8744a7c9" offset="0x00001" loadflag="load16_byte"/>
```

Comment:
We generally use lowercase ROM names, just to simplify things for people with case-sensitive filesystems.  The files will be recognised by CRC inside a zip or 7z archive anyway.